src/components/ImageSlider.vue

```diff
@@ -33,52 +33,55 @@ export default {
         this.currentSlide = this.images.length - 1;
       }
     },
+    selectSlide(index: number): void {
```

Comment:
The `slideStyles` computed property returns an object with a transform property, which is always a string. Instead of returning `{ [key: string]: string }`, you could specify the return type as `{ transform: string }`.